redhat-developer/developers.redhat.com
# [DEVELOPER-3018] Patch content_moderation_notifications module

### Github Issue Link
* #3018 

### Verification Process
1. Create test content moderation role
2. Add user to the new role
3. Create content moderation notification rule using the new role
4. Modify a content node in a way that will send a notification email as per your setup in step 3
5. Check that email is sent successfully to role and author
6. Change user status to "Blocked" for user in new role and author
7. Check that no email is sent to the blocked users
